src/transformers/models/persimmon/modeling_persimmon.py

```diff
@@ -790,11 +790,11 @@ def forward(
         ```python
         >>> from transformers import AutoTokenizer, PersimmonForCausalLM
 
-        >>> model = PersimmonForCausalLM.from_pretrained("ArthurZ/persimmon-8b-base").cuda()
+        >>> model = PersimmonForCausalLM.from_pretrained("ArthurZ/persimmon-8b-base")
```

Comment:
This is not going to work on GPU anyway. 

Also, @sgugger once mentioned to me that, for the usual model docstring CI, we won't put them on GPU. We do this for documentation files `md` only.